cfme/tests/infrastructure/test_host.py

```diff
@@ -535,28 +536,93 @@ def test_add_ipmi_refresh(appliance, setup_provider):
 
 
 @test_requirements.infra_hosts
-@pytest.mark.parametrize("num_hosts", [1, 2, 4])
-@pytest.mark.meta(blockers=[BZ(1634794, forced_streams=["5.10"])], automates=[1634794])
+@pytest.mark.parametrize("num_hosts", [1, ])
 def test_infrastructure_hosts_crud(appliance, setup_provider_min_hosts, provider, num_hosts):
     """
     Polarion:
         assignee: prichard
         casecomponent: Infra
         caseimportance: low
         initialEstimate: 1/6h
-    Bugzilla:
-        1634794
     """
+    # edit host from provider hosts view
     my_slice = slice(0, num_hosts, None)
     hosts_view = navigate_to(provider.collections.hosts, "All")
     for h in hosts_view.entities.get_all(slice=my_slice):
+        hostname = h.name
         h.ensure_checked()
     hosts_view.toolbar.configuration.item_select('Edit Selected items',
                                                 handle_alert=False)
-    try:
-        hosts_view.navigation.select('Compute', 'Infrastructure', 'Providers', handle_alert=False)
-        final_view = provider.create_view(InfraProvidersView)
-        assert final_view.is_displayed
-    except UnexpectedAlertPresentException:
-        pytest.fail("Abandon changes alert displayed, but no changes made.")
-    # TODO add additional crud functionality.
+    edit_view = provider.create_view(HostEditView)
+    stamp = datetime.now()
```

Comment:
I'd use fauxfactory to get a random alphanumeric here, not rely on datetime.

Reply:
Sure, I updated to use fauxfactory. I am just curious as to why this is preferred to datetime. Is it just because datetime will take more processing (CPU cycles)?